orcid-web/src/main/webapp/static/javascript/script.js

```diff
@@ -129,24 +129,29 @@ var OrcidCookie = new function() {
     };
 };
 
-var _gaq = _gaq || [];
-
 var OrcidGA = function() {
     // test and make sure _gaq is working. disconnect.me chrome plugin has
     // caused silent _gaq failures. This check allows us to detect that
     // situation
-    var gaEnabled = false;
-    _gaq.push(function() {
-        gaEnabled = true;
-    });
+    var gaEnabled = true;
+    
 
     this.buildClientString = function(clientGroupName, clientName) {
         return clientGroupName + ' - ' + clientName
     };
     this.gaPush = function(trackArray) {
         if (gaEnabled) {
-            _gaq.push(trackArray);
-            console.log("_gap.push for " + trackArray);
+        	if(typeof trackArray === 'function') {
+        		trackArray();
+        	} else {
+        		if(trackArray[5] == undefined) {
```

Comment:
Look at: 

https://developers.google.com/analytics/devguides/collection/upgrade/reference/gajs-analyticsjs#events

Instead of changing all calls, I keep using the trackArray, but, element at [5] is optional, so, if it is undefined, I just dont send it.